java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterReconciliationController.java

```diff
@@ -116,6 +116,17 @@ private ConfigMapVolumeMount(final String name, final String mountPath, final V1
         }
     }
 
+    private V1PodSpec createPodSpec() {
+        V1PodSpec podSpec = new V1PodSpec();
```

Comment:
Please follow the code style of this project and make all local variables `final` unless they need to be reassigned.

Reply:
👍 I'll move this back into the original method.